project_deadline/views/project_project_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019-2020 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_project_kanban" model="ir.ui.view">
+        <field name="name">view_project_kanban</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//span[@t-if='record.partner_id.value']" position="after">
+                <t
+                    t-if="record.date.raw_value and ((new Date(record.date.raw_value)) lt (new Date()))"
+                    t-set="red"
+                >oe_kanban_text_red</t>
+                <span
+                    style="margin-left: 10px;"
+                    t-attf-class="{{ record.partner_id.value ? red : ''}}"
+                >
+                    <i>
+                        <field name="date" />
+                    </i>
+                </span>
+            </xpath>
+        </field>
+    </record>
+</odoo>
```

Comment:
**issue**:  Tested on runboat. We already have this info in the kanban.

![image](https://github.com/user-attachments/assets/76cbb010-7a91-49ab-86d3-b601afe1853b)

(The date above is added by this module, the two dates below are in the base module)

Looking at the code, even the styling is the same. So we don't need this view at all.

Reply:
Though I saw this information on the kanban view , since in 14.0 date was shown seperately near Partner_id I thought it should be shown in 16.0 even we have start and end date, hence added this.

Removed some of the code as those were not necessary in 16.0 however I misinterpreted this date information behaviour and added here

![image](https://github.com/user-attachments/assets/7fee3699-c10b-4be6-a7de-b4a6fe21f1a9)

Reply:
When project deadline is exceeded, in the base view date color is not changed to "Red" which is different from 14.0 view.

**Suggestion:** Instead of including new field which shows exceeded date in "Red", date already available can be shown in "Red" color.

![image](https://github.com/user-attachments/assets/94498c2a-47bf-483c-a6ed-bd056fb068cb)

Reply:
Latest patchset:

![image](https://github.com/user-attachments/assets/8226ef26-c023-414f-9b23-2710ac21045f)

When partner id is assigned, deadline surpassed will be indicated in "Red"